java/utils.scala

```diff
@@ -213,7 +213,7 @@ object Utils {
     partitions: Int): RDD[Row] = {
 
     var data = rows.map(o => {
-      val r = o.split("\31", -1)
+      val r = o.split("\\|~\\|")
```

Comment:
The `-1` argument is necessary here as otherwise the final record on a line could be dropped, e.g. as in `foo,bar,baz,`.